### ISSUE TITLE
[Bug Report][3.8.7] Tooltip does not close on esc press

### ISSUE THREAD

**Issue by jerosch** [vuetifyjs/vuetify#21501]
### Environment
**Vuetify Version:** 3.8.7
**Vue Version:** 3.5.15
**Browsers:** Microsoft Edge, Mozilla Firefox, Google Chrome
**OS:** Linux x86_64

### Steps to reproduce
Hover over one of the buttons to show the tooltip. Press esc to try to hide the tooltip.

### Expected Behavior
The tooltip closes and is not shown anymore.

### Actual Behavior
The tooltip shows a little bounce animation and remains visible.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNp1kc1uwjAQhF9ltZXKJSSqqv6oTZEq2icot4aDExZhKbFXzhJAiHfHcQjCB272aObz7vj/iK2rsm/mtNsSfmAu1HCthGaFAci7qWIOx3CprBGlDbmLFMRSDHRTsbYWzV8FThbuAGJho1cEDcFOywZ+/+aTAmc/2lEluiNYDP48C4Ar79GULX/e0C9cENqLh99he/SY8alxB3hQ/WNKrPPRI7Cz3MIpMt+sUGqz8r7g8p65bdgaMnJvVh/NoroG6Trz2JtX4uJ6Zeg1AuApwef0JX16xf7wnr5hslZ1S0n0SYO2PANaoIt0)

**Comment by sky0matic** [comment]
This does not respect WCAG guidelines.

From
https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html#dismissible

It mentions :
> 2. Provide a mechanism to easily dismiss the additional content, **such as by pressing Escape**.

**Comment by Harm-Nullix** [comment]
I've got a application under control of an WCAG audit.
This is level AA and I think it should be inmplemented.
It is difficult, because there may be no focus on the tooltip but just the bordered box of the mouse that triggers it.

From the docs:
Tooltip can appear where the cursor is by setting the target prop to cursor. This is currently only available with open-on-click.

This indicates the current tooltip is also not location aware unless the user clicks, which is not enough to be complient!!

So I made this wrapper; feel free to use it, or open a pull request to vuetify to fix this internally:

```
<script setup lang="ts">
  import {onKeyStroke} from '@vueuse/core'

  const show = defineModel<boolean>({default: false})
  onKeyStroke('Escape', (e) => {
    if (!show.value) return true
    e.preventDefault()
    e.stopPropagation()
    show.value = false
  })
</script>

<template>
  <v-tooltip v-bind="$attrs" v-model="show">
    <template v-for="(_, slotName) in $slots" #[slotName]="slotProps">
      <slot :name="slotName" v-bind="slotProps || {}"></slot>
    </template>
  </v-tooltip>
</template>
```

**Comment by J-Sek** [comment]
Wouldn't it be easier to just disable `persistent` via global defaults?
```
defaults: {
  VTooltip: {
     persistent: false,
  },
},
```
hm... it only works with `:persistent="false"` passed as attribute. Not sure why it was done this way.. pretty annoying

**Comment by Harm-Nullix** [comment]
Yes, that is possible!
Cool, I totally forgot to check the VOverlay API and just focused on the VTooltip.
Sorry for that.
I knew in my head about VMenu and VDialog, but didn't remember VTooltip.

To confirm;
Setting as attribute works outofthebox, global not so much (Right now)
To sidetrack, I come across that often, that some values just do not work on the global part.
(Or to let's say: The VBtn within VCardActions or the VSelect within the VTableFooter of VTable, which I know is very specific but I was under the impression the global settings worked that way..)

**Comment by KaelWD** [comment]
Persistent was added in b758c4f7fd73bc31bdde5d47fadcf6d7a01bf0b4 and should have been removed with 9ec452890167d961e619fe29e9bde92ec9de73bd
